lib/client.js

```diff
@@ -1473,6 +1473,54 @@ MatrixClient.prototype.setPresence = function(opts, callback) {
 };
 
 /**
+* This is internal method
+* @param {MatrixClient} client
+* @param {string} opts
+* @param {module:client.callback} callback Optional.
+* @return {module:client.Promise} Resolves: TODO
+* @return {module:http-api.MatrixError} Rejects: with an error response.
+**/
+function _presenceList(callback, client, opts, method) {
+  var path = utils.encodeUri("/presence/list/$userId", {
+      $userId: client.credentials.userId
+  });
+  return client._http.authedRequest(callback, method, path, undefined, opts);
+}
+
+/**
+* Retrieve current user presence list.
+* @param {module:client.callback} callback Optional.
+* @return {module:client.Promise} Resolves: TODO
+* @return {module:http-api.MatrixError} Rejects: with an error response.
+*/
+MatrixClient.prototype.getPresenceList = function(callback) {
+  return _presenceList(callback, this, undefined, "GET");
+}
+
+/**
+* Invite another user to presence list.
+* @param {module:client.callback} callback Optional.
+* @param {string} userId
+* @return {module:client.Promise} Resolves: TODO
+* @return {module:http-api.MatrixError} Rejects: with an error response.
+*/
+MatrixClient.prototype.inviteToPresenceList = function(callback, userId) {
+  let opts = {"invite" : [userId]};
```

Comment:
I'd prefer if the API shape accepted a string array rather than a single user ID. I prefer the array form because it:
 - More accurately represents the underlying HTTP API shape
 - Allows bulk invites/drops in a single round-trip.

Same applies to `:1519`.